src/apis/authApi.js

```diff
@@ -0,0 +1,9 @@
+import axios from './axiosClient';
+
+export const resetPassword = async ({ token, password }) => {
+  const response = await axios.post('/auth/resetPassword', {
+    token,
+    password,
```

Comment:
The backend `resetPassword` endpoint expects a "passwordToken" field and a "newPassword" field. Currently this is passing in a "token" field and a "password" field so it won't be successful: https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/f0cf82f1f4446eaffe3143c03fc372f28d9b8648/src/controllers/authController.js#L184C1-L184C1

Reply:
Thank you , done ,commit  205162a3c8425bacaf3ca25266c1831b5a882052